src/Main.ts

```diff
@@ -127,16 +131,22 @@ window.particlesJS = (tagId: string, params: RecursivePartial<IOptions>) => Part
  * @param pathConfigJson the json path to use in the GET request
  * @param callback the function called after the [[Container]] object is loaded that will be passed as a parameter
  */
-window.particlesJS.load = (tagId: string, pathConfigJson: string, callback: (container: Container) => void) =>
-    ParticlesJS.loadJson(tagId, pathConfigJson, callback);
+window.particlesJS.load = function (tagId: string, pathConfigJson: string, callback: (container: Container) => void) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected call-signature to have a typedef](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5216054)